code/game/objects/items/weaponry.dm

```diff
@@ -617,6 +617,92 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
+	var/reference = FALSE
+	// Chance for above.
+	var/reference_chance = 1
+
+// Coroners are more likely to be evil!
+/obj/item/statuebust/hippocratic/coroner
+	reference_chance = 5
+
+/obj/item/statuebust/hippocratic/Initialize(mapload)
+	. = ..()
+	if(prob(reference_chance))
+		name = "Solemn Vow"
+		desc = "Art lovers will cherish the bust of Hippocrates, commemorating a time when medics still thought doing no harm was a good idea."
+		attack_speed = CLICK_CD_SLOW
+		reference = TRUE
+
+/obj/item/statuebust/hippocratic/examine(mob/user)
+	. = ..()
+	if(reference)
+		. += span_notice("You could activate the bust in-hand to swear or forswear a Hippocratic Oath... but it seems like somebody decided it was more of a Hippocratic Suggestion. This thing is caked with bits of blood and gore.")
+		return
+	. += span_notice("You can activate the bust in-hand to swear or forswear a Hippocratic Oath! This has no effects except pacifism or bragging rights. Does not remove other sources of pacifism. Do not eat.")
+
+/obj/item/statuebust/hippocratic/equipped(mob/living/carbon/human/user, slot)
+	..()
+	if(!(slot & ITEM_SLOT_HANDS))
+		return
+	var/datum/atom_hud/our_hud = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
+	our_hud.show_to(user)
+	ADD_TRAIT(user, TRAIT_MEDICAL_HUD, type)
+
+/obj/item/statuebust/hippocratic/dropped(mob/living/carbon/human/user)
+	..()
+	if(HAS_TRAIT_NOT_FROM(user, TRAIT_MEDICAL_HUD, type))
+		return
+	var/datum/atom_hud/our_hud = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
+	our_hud.hide_from(user)
+	REMOVE_TRAIT(user, TRAIT_MEDICAL_HUD, type)
+
+/obj/item/statuebust/hippocratic/attack_self(mob/user)
+	if(!iscarbon(user))
+		to_chat(user, span_warning("You remember how the Hippocratic Oath specifies 'my fellow human beings' and realize that it's completely meaningless to you."))
+		return
+
+	if(reference)
+		to_chat(user, span_warning("As you prepare yourself to swear the Oath, you realize that doing so on a blood-caked bust is probably not a good idea."))
+		return
+
+	if(HAS_TRAIT_FROM(user, TRAIT_PACIFISM, type))
+		to_chat(user, span_warning("You've already sworn a vow. You start preparing to rescind it..."))
+		if(do_after(user, 5 SECONDS, target = user))
+			user.say("Yeah this Hippopotamus thing isn't working out. I quit!", forced = "hippocratic forswearance")
+			REMOVE_TRAIT(user, TRAIT_PACIFISM, type)
+
+	// they can still do it for rp purposes
+	if(HAS_TRAIT_NOT_FROM(user, TRAIT_PACIFISM, type))
+		to_chat(user, span_warning("You already don't want to harm people, this isn't going to do anything!"))
+
+
+	to_chat(user, span_notice("You remind yourself of the Hippocratic Oath's contents and prepare to swear yourself to it..."))
+	if(do_after(user, 4 SECONDS, target = user))
+		user.say("I swear to fulfill, to the best of my ability and judgment, this covenant:", forced = "hippocratic oath")
```

Comment:
I'm a smidge perplexed, since this can be easily spammed (twice so, with TTS taking a lot more time to recite the paragraphs), unlike the Staff of Asceplius, which oath can't be undone and redone. I'd prefer if these messages were sent to the user only.

Reply:
Ruins the entire point then, I'd rather put it on a cooldown but honestly if someone spams this just punch them and take the bust?

Reply:
a cooldown is fine too.